src/olympia/abuse/templates/abuse/emails/CinderActionRejectVersion.txt

```diff
@@ -6,7 +6,7 @@ Our review found that your content violates the following Mozilla policy or poli
 
 Affected versions: {{ version_list }}
 
-Based on that finding, those versions of your {{ type }} have been disabled on {{ target_url }} and are no longer available for download from Mozilla Add-ons, anywhere in the world. Users who have previously installed those versions will be able to continue using them.
+Based on that finding, those versions of your {{ type }} have been disabled on {{ target_url }} and are no longer available for download from Mozilla Add-ons, anywhere in the world. {% if not is_addon_being_blocked and not target.block %}Users who have previously installed those versions will be able to continue using them.{% endif %}
```

Comment:
I've added a condition on `not target.block` to cover notifications being sent by a rejection if the add-on is _already_ blocked. Technically we don't check whether or not the block that exists covers the affected versions, but since we only use that condition to hide that extra sentence I think it's ok.

Reply:
Copy is being validated by Legal, I'm missing the `{% else %}Users who have previously installed those versions won’t be able to continue using them.{% endif %}`.

Reply:
That'd mean any block on any version of an add-on would lead to the email informing the developer that "Users who have previously installed those versions won’t be able to continue using them", which won't always be correct.

Reply:
True, this only works if we don't have the `else` clause in the mail (we didn't at the time I wrote the patch). I'm going to remove it.

Reply:
https://github.com/mozilla/addons-server/pull/22327/commits/2b7ff0ece9d758ae50e1d53f2ebef430cda81fac